shared_model/interfaces/commands/command.hpp

```diff
@@ -34,48 +34,37 @@ namespace shared_model {
      * General note: this class is container for commands, not a base class, for
      * avoid this misunderstanding class should be final
      */
-    class Command final : public Primitive<Command, iroha::model::Command> {
+    class Command : public Primitive<Command, iroha::model::Command> {
```

Comment:
Not clear for me why final keyword was removed. As the pr comment says commits make consistent Command/Query then why class Query isn't final?

Reply:
Command and Queries are interfaces too. There is no reason to handle variant variable by this class. Thus, from architecture side, we require implementing of Command and Query.